Cargo.toml

```diff
@@ -18,7 +18,8 @@ members = ["examples/*"]
 default = ["tower-log"]
 ws = ["tokio-tungstenite", "sha-1", "base64"]
 multipart = ["multer", "mime"]
-tower-log = ["tower/log"]
+# tower-log = ["tower/log"]
+tower-log = []
```

Comment:
Is this change intentional?

Reply:
Nope 😊 going through everything now making sure its good.

Reply:
Done